bears/xml2/XMLBear.py

```diff
@@ -21,6 +22,21 @@ def path_or_url(xml_dtd):
         return path(xml_dtd)
 
 
+def xml_style_valid(xml_style):
+    """
+    Checks if xml_style is valid.
+
+    :param xml_style: Setting key.
+    :return:          Returns the value if valid else returns none.
+    """
+    style = str(xml_style)
+    if style not in XMLBear._styles:
+        logging.error('Unrecognised style ' + style + '. Valid xml'
```

Comment:
I think this should be a warning and you should mention that you just ignore the argument because you're not erroring out, you're just ignoring that part right?